.vscode/tasks.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I'd rather exclude IDE files from the repo - instead I would update _.gitignore_ file to include _.vscode_ path :)